### PR TITLE
add delete ovs pods after restore nb db

### PR DESCRIPTION
**Pull request by hongzhen-ma**
#### What type of this PR
- Bug fixes
####
add delete ovs pods after restore nb db

#### Which issue(s) this PR fixes:
Fixes #(issue-number)

### CODE REVIEW

File: dist/images/install.sh [hunk 1]
```diff
@@ -3227,6 +3227,13 @@ dbtool(){
           kubectl exec -it -n $KUBE_OVN_NS ${podNameArray[0]} -- mv /etc/ovn/ovnnb_db_standalone.db /etc/ovn/ovnnb_db.db
           kubectl scale deployment -n $KUBE_OVN_NS ovn-central --replicas=$replicas
           echo "finish restore nb db file and ovn-central replicas"
+
+          echo "recreate ovs-ovn pods"
+          podNames=`kubectl get pod -n $KUBE_OVN_NS | grep ovs-ovn | awk '{print $1}'`
```
Comment (oilbeater) [inline dist/images/install.sh:3232]: kubectl delete pod -n $KUBE_OVN_NS -lapp=ovs

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: dist/images/restore-ovn-nb-db.sh [hunk 2]
```diff
@@ -52,3 +52,6 @@ echo "restore nb db file"
 mv /etc/ovn/ovnnb_db_standalone.db /etc/ovn/ovnnb_db.db
 kubectl scale deployment -n $KUBE_OVN_NS ovn-central --replicas=$replicas
 echo "finish restore nb db file and ovn-central replicas"
+
+echo "recreate ovs-ovn pods"
+kubectl delete pod -n $KUBE_OVN_NS -l app=ovs
```
Comment (hongzhen-ma) [inline dist/images/restore-ovn-nb-db.sh:57]: 镜像中应该删掉了kubectl的文件，脚本执行还要依赖这个

File: dist/images/install.sh (hunk 1 above)
Comment (hongzhen-ma) [reply dist/images/install.sh]: modified

File: dist/images/restore-ovn-nb-db.sh (hunk 2 above)
Comment (oilbeater) [reply dist/images/restore-ovn-nb-db.sh:57]: 镜像我修改一下

**Review by oilbeater — APPROVED** [review]

**Comment by hongzhen-ma** [comment]
修改已回合release-1.8 和 release-1.9 分支

**Pull request merged**